spec/models/donor_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+describe Donor do
+  context ".primary_address" do
+    it "returns the first address" do
+      donor1 = donors(:starfleet_command)
```

Comment:
`aggregate_failures` in this spec

Reply:
not something I am familiar with or we currently do in this stack. Let's look at this for an improvement ticket. I would like to get some work merged short term.